backend/dataline/services/llm_flow/toolkit.py

```diff
@@ -64,9 +64,15 @@ def execute_sql_query(
     db: SQLDatabase, query: str, for_chart: bool = False, chart_type: Optional[ChartType] = None
 ) -> QueryRunData:
     """Execute the SQL query and return the results or an error message."""
-    result = cast(Result[Any], db.run(query, fetch="cursor", include_columns=True))  # type: ignore[misc]
+    if db.dialect == "mssql":
+        with db._engine.begin() as connection:
```

Comment:
Check if we can add this logic inside the SQLDatabase run method. The run method should worry about how it runs the query. I don't want to duplicate logic unless necessary.

Also we should include the views in the setup, I don't see any reason why not to

Reply:
Should we include the views for all db types?

Reply:
I think so, would be nice. Any downsides you can think of?

Reply:
If we can run sql statements against views, I don't see any issues with adding them!